Library/Formula/avidemux.rb

```diff
@@ -1,15 +1,23 @@
-# Help! Wanted: someone who can get Avidemux working with SDL.
 class Avidemux < Formula
   homepage "http://fixounet.free.fr/avidemux/"
-  url "https://downloads.sourceforge.net/avidemux/avidemux_2.6.8.tar.gz"
-  sha256 "02998c235a89894d184d745c94cac37b78bc20e9eb44b318ee2bb83f2507e682"
-  revision 1
+
+  stable do
+    url "https://downloads.sourceforge.net/avidemux/avidemux_2.6.8.tar.gz"
+    sha256 "02998c235a89894d184d745c94cac37b78bc20e9eb44b318ee2bb83f2507e682"
+
+    # remove ffmpeg binary from targets (fixed upstream)
+    # https://github.com/mean00/avidemux2/commit/bf018562bc6cff300edb1e59ee060a10df902bdc
+    # http://avidemux.org/smuf/index.php/topic,16379.15.html
+    patch :DATA
```

Comment:
Does `https://github.com/mean00/avidemux2/commit/bf0185.diff` apply cleanly? If so, let's use that instead.